openmp issue in when compiling from pip

I had an issue when installing mdtraj through pip:
```
mdtraj/rmsd/src/center.c: In function ‘inplace_center_and_trace_atom_major’:
  mdtraj/rmsd/src/center.c:22:13: error: ‘n_frames’ not specified in enclosing ‘parallel’
     22 |     #pragma omp parallel for default(none) shared(coords, traces) \
        |             ^~~
  mdtraj/rmsd/src/center.c:22:13: error: enclosing ‘parallel’
  mdtraj/rmsd/src/center.c:27:27: error: ‘n_atoms’ not specified in enclosing ‘parallel’
     27 |         confp = &coords[k * n_atoms * 3];
        |                         ~~^~~~~~~~~
  mdtraj/rmsd/src/center.c:22:13: error: enclosing ‘parallel’
     22 |     #pragma omp parallel for default(none) shared(coords, traces) \
        |             ^~~
  error: command 'gcc' failed with exit status 1
  ----------------------------------------
  ERROR: Failed building wheel for mdtraj
  Running setup.py clean for mdtraj
Failed to build mdtraj
ERROR: Could not build wheels for mdtraj which use PEP 517 and cannot be installed directly
```

I have the following:
* python 3.7.4
* gcc 9.1.0
* cython 0.29.13
* openmp 8.0.1-1

I tried to do
```
pip install --no-use-pep517 mdtraj
```
but it didn't help.

In the end I managed to compile and install mdtraj by modifying directly `mdtraj/rmsd/src/center.c` at line 22:
```c
#pragma omp parallel for default(none) shared(coords, traces, n_frames, n_atoms)
```
I don't know if it fixed the underling issue but I am able to load trajectories and work with it. I hope it can help :).

since that PR, my builds started failing with:
```
  mdtraj/rmsd/src/center.c: In function ‘inplace_center_and_trace_atom_major’:
  mdtraj/rmsd/src/center.c:22:50: error: ‘n_frames’ is predetermined ‘shared’ for ‘shared’
       #pragma omp parallel for default(none) shared(coords, traces, n_frames, n_atoms) \
                                                    ^
  mdtraj/rmsd/src/center.c:22:50: error: ‘n_atoms’ is predetermined ‘shared’ for ‘shared’
  error: command 'gcc' failed with exit status 1
  ----------------------------------------
  ERROR: Failed building wheel for mdtraj
Failed to build mdtraj
```

--- a/mdtraj/rmsd/src/center.c
+++ b/mdtraj/rmsd/src/center.c
@@ -19,7 +19,7 @@ void inplace_center_and_trace_atom_major(float* coords, float* traces, const int
     __m128 x, y, z, x2, y2, z2;
 
     #ifdef _OPENMP
-    #pragma omp parallel for default(none) shared(coords, traces, n_frames, n_atoms) \
+    #pragma omp parallel for shared(coords, traces) \
         private(sx_, sy_, sz_, trace_, mux_, muy_, muz_, sxf, syf, szf, \
         confp, i, x, y, z, x2, y2, z2, sx, sy, sz, trace)
     #endif

